macros/core/PGbasicmacros.pl

```diff
@@ -1773,7 +1773,7 @@ sub openDivSpan {
 		return ();
 	}
 	my $option_ref = {};
```

Comment:
This needs to be
```suggestion
	my $option_ref   = {};
```
for perltidy.  This is needed because you added the `= ''` on the line below, and so now this needs to line up with that.